Tests/EmbraceCoreTests/Session/SessionControllerTests.swift

```diff
@@ -89,30 +89,6 @@ final class SessionControllerTests: XCTestCase {
         XCTAssertTrue(controller.currentSession!.coldStart)
     }
 
-    func test_startSession_ifStartAtMatchesAllowedColdStartInterval_marksSessionAsColdStartTrue() throws {
```

Comment:
Can you add tests for the new cold start logic? Should verify that the first session only is marked as `cold_start = true`